napari_aicssegmentation/widgets/collapsible_box.py

```diff
@@ -18,14 +18,14 @@
 
 
 class CollapsibleBox(QWidget):
```

Comment:
Let's have a simple description / javadoc for the Widget. In general i'd like to see a description comment for all classes and public methods - always helpful when working in the IDE. Even if the name of the class is obvious.

Reply:
Does what I have in lines 9-17 work? Maybe I put it in the wrong place?

Reply:
I moved the doc block down into the class :)